ansible_collections/arista/avd/roles/eos_designs/templates/network_services/ethernet-interfaces.j2

```diff
@@ -67,11 +67,79 @@ ethernet_interfaces:
 {%         endfor %}
 {%     endfor %}
 {% endfor %}
+{% set point_to_point_service_subif_parents = [] %}
+{% set point_to_point_service_port_channel_members = [] %}
+{# pseudowire interfaces #}
+{% for tenant in switch.tenants | arista.avd.natural_sort %}
+{%     for point_to_point_service in tenants[tenant].point_to_point_services | arista.avd.default([]) %}
```

Comment:
refactor to parse the smaller list you set in logic.j2

Reply:
Fixed. Also fixed this in patch-panel.j2 and router-bgp-vpws.j2 templates.